pages/in/tokensale/index.vue

```diff
@@ -544,14 +574,18 @@ export default {
   }
 }
 
-.tokensale-progress-wrapper {
+.tokensale-presale-wrapper, .tokensale-progress-wrapper {
   padding-right: 0;
   padding-left: 0;
 
   > div {
     margin-right: -8px;
     margin-left: -8px;
   }
```

Comment:
```css
 padding-right: 0;
 padding-left: 0;
 
 > div {
   margin-right: -8px;
   margin-left: -8px;
 }
```
The above CSS is for achieving overlapping appearance of the progress bar only
I think `tokensale-presale-wrapper` class does not need this in order to preserve default padding